### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by TammyChen21**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,14 +26,15 @@ <h1>Quiz App</h1>
 
     <div class="card">
```
Comment (lenzloesche) [inline index.html:27]: card could be a section

File: index.html [hunk 2]
```diff
@@ -26,14 +26,15 @@ <h1>Quiz App</h1>
...
       <p>Lorim Impsum dolor sit?</p>
-      <button>Show Answer</button>
+      <button class="answer">Show Answer</button>
     </div>
 
     <fotter>
       <ul id="foot">
-        <li></li>
-        <li><img src="img/lesezeichen" alt="bild2" /></li>
```
Comment (lenzloesche) [inline index.html:35]: alt should have a value that can be understood like: lesezeichen.

File: index.html [hunk 3]
```diff
@@ -26,14 +26,15 @@ <h1>Quiz App</h1>
 
     <div class="card">
       <p>Lorim Impsum dolor sit?</p>
-      <button>Show Answer</button>
+      <button class="answer">Show Answer</button>
     </div>
 
     <fotter>
```
Comment (lenzloesche) [inline index.html:32]: small spelling mistake: `footer` not `fotter`

**Review by lenzloesche — COMMENTED** [review]
well done!
hope the comments help
maybe add Vercel, so we can preview the site easily.

**Comment by TammyChen21** [comment]
> well done!
> hope the comments help
> maybe add Vercel, so we can preview the site easily.

the comments are very helpful, thanks very much

**Pull request merged**